extensions/plugins/system/redcore/redcore.php

```diff
@@ -119,6 +119,8 @@ public function onAfterInitialise()
 							'absoluteHrefs'     => $input->get->getBool('absoluteHrefs', true),
 						);
 
+						$this->cleanCache();
```

Comment:
then you could also go for a https://github.com/redCOMPONENT-COM/redCORE/blob/develop/extensions/libraries/redcore/api/hal/hal.php#L226 header check, something like X-WEBSERVICE-STATEFULL and add it into documentation here: https://github.com/redCOMPONENT-COM/redCORE/blob/develop/extensions/libraries/redcore/layouts/webservice/documentation.php#L164 because atm this feature would be invisible....